utils/flare.py

```diff
@@ -215,13 +216,14 @@ def _add_logs_tar(self):
         self._add_log_file_tar(self._forwarder_log)
         self._add_log_file_tar(self._dogstatsd_log)
         self._add_log_file_tar(self._jmxfetch_log)
+        self._add_log_file_tar(self._gometro_log, warn=False)
         self._add_log_file_tar(
             "{0}/*supervisord.log".format(os.path.dirname(self._collector_log))
         )
 
-    def _add_log_file_tar(self, file_path):
+    def _add_log_file_tar(self, file_path, warn=True):
         for f in glob.glob('{0}*'.format(file_path)):
```

Comment:
actually the `glob.glob` step here prevents us from adding a log file that doesn't exist, so you can ignore my comment on the warnings and revert these changes, thanks!